autospec/specfiles.py

```diff
@@ -1641,7 +1641,8 @@ def write_gradle_pattern(self):
             self._write_strip("pushd " + self.subdir)
 
         # Point to our local maven repo, first
-        self._write_strip("sed -i 's/\(repositories\s*{\)/\\1\\n    mavenLocal()/' build.gradle")
```

Comment:
We need to add other files like:

sed -i 's/\(repositories\s*{\)/\1\n mavenLocal()/' settings.gradle
sed -i 's/\(repositories\s*{\)/\1\n mavenLocal()/' build.gradle.kts
sed -i 's/\(repositories\s*{\)/\1\n mavenLocal()/' settings.gradle.kts
sed -i 's/\(repositories\s*{\)/\1\n mavenLocal()/' buildSrc/build.gradle.kts
sed -i 's/\(repositories\s*{\)/\1\n mavenLocal()/' buildSrc/settings.gradle.kts

And also, assume that all of them can fail, cause not all of them exists

Reply:
Is that a fixed list, or should I do something like:
`find . -type f '(' -name '*.gradle' -o -name '*.gradle.kts' ')' -exec sed -i ...`

Reply:
thats a better option